packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -122,50 +122,57 @@ class InteractionResponses {
   }
 
   /**
-   * Fetches the initial reply to this interaction.
+   * Fetches a reply to this interaction.
    * @see Webhook#fetchMessage
+   * @param {MessageResolvable|'@original'} [message='@original'] The response to fetch
```

Comment:
This would actually be `Snowflake|'@original'`, because `Webhook#fetchMessage` doesn't support message resolvables:

https://github.com/discordjs/discord.js/blob/68d5712deae85532604d93b4505f0953d664cde7/packages/discord.js/src/structures/Webhook.js#L283-L292

Reply:
This could be edited in `Webhook#fetchMessage` to support `MessageResolvable` just like the other two methods do:

https://github.com/discordjs/discord.js/blob/68d5712deae85532604d93b4505f0953d664cde7/packages/discord.js/src/structures/Webhook.js#L321
https://github.com/discordjs/discord.js/blob/68d5712deae85532604d93b4505f0953d664cde7/packages/discord.js/src/structures/Webhook.js#L365

Although there wouldn't be much of a point to fetch an already-fetched message, it can serve to refresh the data.